src/components/Header/Header.js

```diff
@@ -203,16 +203,6 @@ function GardenNavItems({ garden }) {
           Wiki
         </Link>
       )}
-      <Link
```

Comment:
Remove the documentation link because there is a link on the home screen and the header was already convoluted enough.